docker/Dockerfile.dev

```diff
@@ -0,0 +1,24 @@
+FROM node:14
```

Comment:
Hah, a delight! All my work Docker images have to fuse Node and Python base images together (we have Good Reasons Actually for serving our static assets from our backend).

Reply:
ugggggg that reminds me, would we consider separating static assets into a webpack bundle and pushing them to S3/CloudFront?  probably not something we need to do at current scale

Reply:
Yeah, that's a Future Problem.

Reply:
Definitely a Future Problem, but a solution I'm not opposed to. I think it'll be waiting for the transition away from the model/controller handling so much of the view functionality.